src/api.js

```diff
@@ -174,6 +174,13 @@ export const MOCK_EVENT = {
 	yes_rsvp_count: 23
 };
 
+export const MOCK_BEST_TOPIC = {
```

Comment:
Another option -- since this kind of 'best topic' topic appears distinguished a little from top-level topics, does it make sense to put it directly on `MOCK_CATEGORY` instead...?

```
export const MOCK_CATEGORY = {
    best_topics: [{
	id: 16645,
	name: 'Volunteering',
	urlkey: 'volunteering',
	lang: 'en_US'
    }]
};
```
I'm guessing if we want a `const MOCK_TOPIC`we'll want it to look exactly like what https://www.meetup.com/meetup_api/docs/topics` returns.